.circleci/generate.sh

```diff
@@ -33,6 +37,10 @@ print_help()
   echo "   -a Generate the default config.yml using low resources and the three templates"
   echo "      (config.yml.LOWRES, config.yml.MIDRES and config.yml.HIGHRES). Use this for"
   echo "      permanent changes in config-2_1.yml that will be committed to the main repo."
+  echo "   -b Specify the base git branch for comparison when determining changed tests to"
+  echo "      repeat. You can also set the BASE_BRANCH environment variable to control this."
+  echo "      Defaults to ${BASE_BRANCH}. Note that this option is not used when the '-a' flag"
+  echo "      is specified."
```

Comment:
I'd put this below the -l/-m/-h flags, so we have the closely related -a/-l/-m/-h together.

Reply:
Sure, will move